services/app-graphql/src/schema.graphql

```diff
@@ -2020,6 +2029,9 @@ type RateStripped {
 
     "The latest submitted rate revision"
     latestSubmittedRevision: RateRevisionStripped!
+
+    "Contracts this rate is related to"
```

Comment:
suggestion: be explicit here in comment  -"related" means  what. It means both the parent contract and the additional linked contracts using the rate